dunamai/__init__.py

```diff
@@ -990,29 +991,62 @@ def get_version(
     :param third_choice: Callback to determine a version if the installed
         package cannot be found by name.
     :param fallback: If no other matches found, use this version.
+    :param ignore: Ignore this version if it is found.
```

Comment:
Could you explain your use case for `ignore`? I'm not quite sure when you'd need it, and I'd like to understand better before adding it.

Reply:
When working in an environment where both the package is installed and in development you have multiple versions. The installed version has v0. Cause that is not the version I want I would filter it out.

Reply:
Gotcha, that makes sense.

I did just think of one thing that's a bit of a wrinkle here. For example:

* first_choice: `Version.from_any_vcs`
* ignore: `["0.1.0"]`

Even if `Version.from_any_vcs` finds 01.0, they'll compare not equal because the found version will have `commit="abc"` and the ignored version will have `commit=None`. I think we'll need some special comparison logic where, if a field in the ignored version is None, then that field is ignored for the comparison.

Reply:
One other thought: Allowing a plain string in the `ignore` list could be ambiguous whether to handle it as `Version(x)` or `Version.parse(x)` (from your #41). It might be better to simply have `ignore: Sequence[Version]` and let the caller choose when creating the ignore list.

Reply:
Good catch. I proposed a spacial comparison, but not sure if that is what we are looking for.